net-p2p/bitcoin-core/bitcoin-core-26.0.ebuild

```diff
@@ -0,0 +1,415 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit autotools bash-completion-r1 check-reqs db-use desktop edo multiprocessing python-any-r1 systemd toolchain-funcs xdg-utils
+
+DESCRIPTION="Reference implementation of the Bitcoin cryptocurrency"
+HOMEPAGE="https://bitcoincore.org/"
+SRC_URI="https://github.com/bitcoin/bitcoin/archive/v${PV}.tar.gz -> ${P/-core}.tar.gz"
+S="${WORKDIR}/${P/-core}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 ~arm ~arm64 ~ppc ~ppc64 x86 ~amd64-linux ~x86-linux"
```

Comment:
This is a straight to stable commit. I will fix this on merge. But please tests your ebuilds with pkgcheck, which would have shown that.

Reply:
Oops. Why wasn't that caught by the CI bot here?

> please tests your ebuilds with pkgcheck

I wish I could, but I honestly can't figure out how to run it.
```
$ pkgcheck scan net-p2p/bitcoin-core/bi
tcoin-core-26.0.ebuild
pkgcheck: error: failed parsing 'make.defaults' in '/etc/portage/profile': [Errno 2
1] Is a directory: '/etc/portage/profile/make.defaults'
```
Yes, no shit that's a directory. So? That's not supported? Crap tool. 👎🏻